src/utils/search.py

```diff
@@ -1,17 +1,19 @@
+"""
+Search
+"""
 import csv
 
-import torch
+# import torch
```

Comment:
If torch is not needed, the import should be removed instead of commented

Reply:
Indeed